Resources/Prototypes/Entities/Structures/Walls/walls.yml

```diff
@@ -712,4 +712,30 @@
       - FullTileMask
       layer:
       - GlassLayer
-  - type: Airtight
\ No newline at end of file
+  - type: Airtight
+
+- type: entity
+  id: WallForce
+  name: Force Wall
+  components:
+    - type: Tag
+      tags:
+        - Wall
+    - type: Physics
+      bodyType: Static
+    - type: Fixtures
+      fixtures:
+        - shape:
+            !type:PhysShapeAabb
+            bounds: "-0.5,-0.5,0.5,0.5"
+          layer:
+            - GlassLayer
+          mask:
+            - FullTileMask
```

Comment:
```suggestion
          mask:
            - FullTileMask
          layer:
            - GlassLayer
```
Flip these to be consistent with all other ones.

GlassLayer allows LaserGuns to shoot through. Not sure if you want that. If not then change it to WallLayer.

Reply:
I did some testing and I can't have it included on the prototype.

if I have it included on the prototype, they won't despawn at all: 
![image](https://user-images.githubusercontent.com/54602815/168441394-e32efd6e-31da-47ec-8678-fc2fbbde9ea8.png)

However if I have it as before (with active walls and ensuring comp) then the walls actually despawn.

Reply:
Nevermind on that. Was a one off bug or something. Fixed.